blockbot-api/src/main/java/io/github/quiltservertools/blockbotapi/mixin/TellRawCommandMixin.java

```diff
@@ -23,7 +24,8 @@ public abstract class TellRawCommandMixin {
         at = @At(value = "HEAD")
     )
     private static void relayMeToDiscord(CommandContext<ServerCommandSource> context, CallbackInfoReturnable<Integer> cir) throws CommandSyntaxException {
-        if (context.getInput().replace("/tellraw ", "").startsWith("@a")) {
+        // Check if tellraw is sent to every online player
+        if (EntityArgumentType.getPlayers(context, "targets").size() == context.getSource().getPlayerNames().size()) {
```

Comment:
Feels like a weird workaround, maybe it would be possible to check if the `EntitySelector` argument only targets a specific player in the future. I think it should also be possible to disable tellraw relaying (if it isn't already).

Reply:
I added another commit that covers all cases that I know of (public would be relayed, private wouldn't):
`/tellraw DrexHD [{"text":"private"}]`
`/tellraw @s [{"text":"private"}]`
`/tellraw @p [{"text":"private"}]`
`/tellraw @r [{"text":"private"}]`
`/tellraw @a [{"text":"public"}]`
`/tellraw @e[type=player] [{"text":"public"}]`

`/tellraw @a[distance=..100] [{"text":"private/public"}]` (if everyone received the message (everyone was in range), it would get relayed, otherwise it wont)